### ISSUE TITLE
[BUG] `Chart(fullscreen=True)` needs `Chart(screen=n)`

### ISSUE THREAD

**Issue by jlixfeld** [louisnw01/lightweight-charts-python#165]
Assuming the library's intention is for `Chart(maximize=True)` and `Chart(screen=n)` to be mutually exclusive, setting the former without setting the latter should not raise an error.

In practise, `AttributeError` is thrown if `screen` is not set and `maximize` is set:

```
chart = Chart(toolbox=True, inner_width=0.8, inner_height=1, maximize=True)
```

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/usr/local/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/jlixfeld/Library/Caches/pypoetry/virtualenvs/ariobot-OV7uRE50-py3.11/lib/python3.11/site-packages/lightweight_charts/chart.py", line 28, in __init__
    self.create_window(width, height, x, y, screen, on_top, maximize)
  File "/Users/jlixfeld/Library/Caches/pypoetry/virtualenvs/ariobot-OV7uRE50-py3.11/lib/python3.11/site-packages/lightweight_charts/chart.py", line 37, in create_window
    width, height = screen.width, screen.height
                    ^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'width'
```
The error is thrown as a result of this code block:
https://vscode.dev/github/louisnw01/lightweight-charts-python/blob/main/lightweight_charts/chart.py#L35-L37
```
        screen = webview.screens[screen] if screen is not None else None
        if maximize:
            width, height = screen.width, screen.height
```

**Comment by louisnw01** [comment]
Hey @jlixfeld,

Thanks for pointing this out; Another issue related to screens was raised previously, regarding the default behaviour when no screen is set, so the `screen=None` part will have to stay. However I realise that this is also a major issue, so i'll look into an alternative way to fix this.

Louis

**Comment by jlixfeld** [comment]
Fair enough.  I don't think this is a major issue.  Maybe it's reasonable to raise an exception if the conditions are met, otherwise, do you see any issue limiting the scope to the method, and adjust the test to set a value?

```python
def create_window(self, width, height, x, y, screen=None, on_top=False, maximize=False):
  if maximize:
    if not screen:
      raise ValueError('when using maximize, you must specify the screen argument')
  else:
    screen = webview.screen[screen]
```

or 

```python
def create_window(self, width, height, x, y, screen=None, on_top=False, maximize=False):
  if maximize:
    if not screen:
      screen = webview.screen[0]
  else:
    screen = webview.screen[screen]
```

**Comment by louisnw01** [comment]
Hey @jlixfeld,

Fixed in the latest version (1.0.18.2).

Louis
